src/client/core/utils/array.js

```diff
@@ -66,3 +66,22 @@ export function remove (arr, item) {
     if (index > -1)
         nativeSplice.call(arr, index, 1);
 }
+
+export function equals (arr1, arr2) {
+    if (!arr2)
+        return false;
+
+    if (arr1.length !== arr2.length)
+        return false;
+
+    for (var i = 0, l = arr1.length; i < l; i++) {
+        if (arr1[i] instanceof Array && arr2[i] instanceof Array) {
```

Comment:
We should check whether target element is changed position in the DOM. If it changes parent, but still on previous place, click doesn't raised natively in all browser (exc. Firefox)